README.md

```diff
@@ -106,6 +106,14 @@ rosdep install --from-paths src --ignore-src -r -y
 > poetry install --with openset,nomad
 > ```
 
+#### 1.4 Configure RAI
+
+Run the configuration tool to set up your vendor and other settings:
+
+```bash
+streamlit run src/rai/rai/utils/configurator.py
+```
+
```

Comment:
Add a sentence that user should go to the browser and open localhost:8501

Reply:
It opens automatically. The port is unknown before running the script.